### ISSUE TITLE
Create document via inputstream results in blank page

### ISSUE THREAD

**Issue by 1nutzername** [aspose-words/Aspose.Words-for-Java#59]
I saved my docx documents temporarily as blob in a postgresql database using jdbc. After fetching and initializing them with help of the Document::constructor (Aspose 19.10) all of the created documents are blank.
### Example code
```java 
List<Document> test(List<InputStream> streams) {
   List<Document> a = new LinkedList<>();
   for(InputStream stream: streams) {
      a.add(new Document(stream);
   }
   return a;
}
```

### The cause
The Document::constructor for inputstreams calls `zz51.zzY(stream)` and this method contains the following lines:
``` java
if (var0 instanceof FileInputStream) {
    return new zz1K((FileInputStream)var0);
}
if (var0.markSupported()) {
    return new zz1J(var0);
}

return zzX(var0);
```
My InputStream isn't of type FileInputStream but it supports marks and in my case it jumps into _zz1J_. The following snipped is the cause of my blank documents:
```
this.zzSv = (long)var1.available();
this.zz2Y = false;
this.zz2Z.mark((int)this.zzSv);
```
According the official javadoc its not recommended to do that ([Javadoc InputStream](https://docs.oracle.com/javase/8/docs/api/?java/io/InputStream.html)) because "some implementations of InputStream will return the total number of bytes in the stream, **many will not**". So doesn't the _BlobInputStream_ of the Postgresql-jdbc implementation which returns the default value of InputStream::available [(Postgresql BlobInputStream)](https://github.com/pgjdbc/pgjdbc/blob/5e48c4d0714fd656fb1997f29c1fa2d4fee60708/pgjdbc/src/main/java/org/postgresql/largeobject/BlobInputStream.java).

### Solution/Workaround
My workaround is to wrap the InputStream into a class that extands from InputStream and override the methods returning false in the method `markSupported` to skip the jump into _zz1J_. Instead it reads the stream as it is in _zzX_.

br

**Comment by AlexNosk** [comment]
@i7i5 Thank you for reporting this to us. I logged the issue WORDSJAVA-2248 in our bug tracking system. We will let you know once it is resolved.

**Comment by AlexNosk** [comment]
@i7i5 The issue you have reported earlier is fixed. The fix is included into the latest 20.01 release of Aspose.Words for Java. You can download this version from [here](https://downloads.aspose.com/words/java).
